src/handlers/handlers.js

```diff
@@ -0,0 +1,17 @@
+var fs = require('fs');
+var path = require('path');
+
+var handlers = (module.exports = {});
+
+handlers.serveLanding = function(request, response) {
+  fs.readFile(
+    path.join(__dirname, '..', '..', 'public', 'index.html'),
+    function(err, file) {
+      if (err) {
+        throw err;
```

Comment:
Was just including the ref to foundersandcoders/master-reference#456 as reference, not expecting that you'd read it. Anyway, fair to say there's not yet consensus on the branch thing. I'd be happy to merge this into `mentor-walkthrough` for now, I just wanted to raise it with @jsms90 @des-des and @bradreeder.

I agree that lock files are part of a node project. I was asking whether we really needed to make each step-X directory a fully fledged node project; my understanding was that they were just reference examples. If that's true, we'd just need the HTML/CSS/JS. "other stuff" just means the `package.json`, lol.